docs/_docs/admin-guide/tavern.md

```diff
@@ -0,0 +1,379 @@
+2023/02/24 01:02:37 [WARN] Test data is enabled
+2023/02/24 01:02:37 Starting HTTP server on 0.0.0.0:80
+```
+
+### PPROF
+
+Running Tavern with the `ENABLE_PPROF` environment variable set will enable performance profiling information to be collected and accessible. This should never be set for a production deployment as it will be unauthenticated and may provide access to sensitive information, it is intended for development purposes only. Read more on how to use `pprof` with tavern in the [Developer Guide](/dev-guide/tavern#performance-profiling).
+
+## Build and publish tavern container
+
+If you want to deploy tavern without using the published version you'll have to build and publish your own container.
+
+### Build your container
+
+```bash
+cd ./realm
+docker build --tag tavern:dev --file ./docker/tavern.Dockerfile .
+```
+
+### Publish your container to docker hub
+
+If you haven't before [sign-up for a docker hub account](https://hub.docker.com/signup) and login with the CLI `docker login`
+
+```bash
+docker tag tavern:dev <YOUR_DOCKER_HUB_USERNAME>/tavern:dev
+docker push <YOUR_DOCKER_HUB_USERNAME>/tavern:dev
+```
+
+### Specify your container during terraform deploy
+
+```bash
+terraform apply -var="gcp_project=<PROJECT_ID>" -var="oauth_client_id=<OAUTH_CLIENT_ID>" -var="oauth_client_secret=<OAUTH_CLIENT_SECRET>" -var="oauth_domain=<OAUTH_DOMAIN>" -var="tavern_container_image=<YOUR_DOCKER_HUB_USERNAME>/tavern:dev"
+```
+
+## User Interface
+
+## GraphQL API
+
+### Playground
+
+If you'd like to explore the Graph API and try out some queries, head to the `/graphiql` endpoint of your Tavern deployment. This endpoint exposes an interactive playground for you to experiment with GraphQL queries. Currently, this is used to fill gaps between developed backend functionality and in-development frontend functionality.
+
+![/assets/img/tavern/graphiql.png](/assets/img/tavern/graphiql.png)
+
+#### Some sample queries to get started
+
+##### List all beacons
+
+```graphql
+query get_beacons {
+  beacons {
+    id
+    identifier
+    name
+    hostPlatform
+    hostIdentifier
+    hostPrimaryIP
+  }
+}
+```
+
+##### Create a tome
+
+```graphql
+mutation CreateTome ($input: CreateTomeInput!) {
+  createTome(input: $input) {
+    id
+    name
+  }
+}
+```
+
+```json
+{
+  "input": {
+    "name": "Test tome",
+    "description": "Just a sample",
+    "eldritch": "print(input_params['print_string'])",
+    "fileIDs": [],
+    "paramDefs": "[{\"name\":\"print_string\",\"label\":\"Print String\",\"type\":\"string\",\"placeholder\":\"A message to print\"}]"
+  }
+}
+```
+
+##### Create a task
+
+```graphql
+mutation createQuest($input: CreateQuestInput!, $beaconIDs:[ID!]!){
+  createQuest(input: $input, beaconIDs: $beaconIDs) {
+    id
+  }
+}
+```
+
+```json
+{
+  "input": {
+    "name": "Run test tome",
+    "tomeID": "21474836488",
+    "parameters": "{\"print_string\":\"Hello World\"}"
+  },
+  "sess": ["8589934593"]
```

Comment:
I think this needs to be updated to `"beaconIDs"`
<img width="214" alt="image" src="https://github.com/spellshift/realm/assets/7121375/d38281cc-4e75-4649-aa78-e3e2a90db60b">